### PR TITLE
Update to the latest upstream version 20190115.

### PR DESCRIPTION
**Pull request by mkosola**

### CODE REVIEW

**Comment by saukko** [comment]
Maybe just remove the .ksh and .csh files for file test. AutoReq: no is also one option, but that might be a bit much, need to think a bit further on this.

**Comment by mkosola** [comment]
Updated patch to only remove .ksh and .csh files.

File: rpm/0001-Remove-unsupported-csh-and-ksh.patch [hunk 1]
```diff
@@ -0,0 +1,36 @@
+From 2ccaa3fd65388deccd45afbc7f4c5ddb33491587 Mon Sep 17 00:00:00 2001
+From: Matti Kosola <matti.kosola@jolla.com>
+Date: Tue, 26 Feb 2019 02:14:09 -0500
+Subject: [PATCH] Remove unsupported csh and ksh
+
```
Comment (saukko) [inline rpm/0001-Remove-unsupported-csh-and-ksh.patch:5]: We should add comment here that this is done as having these files will automatically add dependency to the csh and ksh that we do not have and thus the ltp can't be installed.

**Comment by saukko** [comment]
LGTM

**Pull request merged**